README.md

```diff
@@ -1,2 +1,67 @@
 # rumal_docker
 A docker container with a ready to run rumal instance
+
+
+# Installation 
+
+System-wide requirements
+* Docker 
+* git
+
+    
+**Docker:**
+
+You will need docker to be able to build dockefiles, follow the instructions on [this link](https://docs.docker.com/engine/installation/) on your respective system. You will need to be able to run the command ```docker-compose```.
+
+
+**Cloning and building Dockerfile**
+```sh
+$ git clone https://github.com/Dennisparchkov/rumal_docker.git
```

Comment:
Please fix the URL using the official repo https://github.com/thugs-rumal/rumal_docker.git